### PR TITLE
Update notification

### PR DESCRIPTION
**Pull request by intern-tieumomo**
## WHAT this PR does?
Show notification

## Notes Impacted Areas:
- N/A.

## Attachments:
- 
Screenshot from 2020-09-24 13-19-28
Screenshot from 2020-09-24 13-19-08
Screenshot from 2020-09-24 13-15-55

### CODE REVIEW

**Comment by intern-tieumomo** [comment]
ready

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <link rel="stylesheet" href="{{ asset('css/plugins.css') }}">
     <link rel="stylesheet" href="{{ asset('css/helper.css') }}">
     <link rel="stylesheet" href="{{ asset('css/style.css') }}">
+    <link rel="stylesheet" href="bower_components/toastr/toastr.min.css">
```
Comment (thaonp104) [inline resources/views/layouts/app.blade.php]: cho vào assert()

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -156,8 +157,12 @@
 <script src="{{ asset('js/vendor/jquery-1.12.4.min.js') }}"></script>
 <script src="{{ asset('js/popper.min.js') }}"></script>
 <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+<script src="bower_components/toastr/toastr.min.js"></script>
+<script src="bower_components/sweetalert2/dist/sweetalert2.all.js"></script>
```
Comment (thaonp104) [inline resources/views/layouts/app.blade.php]: tương tự

**Comment by intern-tieumomo** [comment]
ready

**Pull request merged**